src/externalized/DefaultContentManager.cc

```diff
@@ -854,7 +859,7 @@ const std::map<uint16_t, uint16_t>& DefaultContentManager::getMapItemReplacement
 	return m_mapItemReplacements;
 }
 
-const DealerInventory* DefaultContentManager::getDealerInventory(int dealerId) const
+const DealerInventory* DefaultContentManager::getDealerInventory(ArmsDealerID dealerId) const
```

Comment:
If getDealer range checks its argument I think this method should also do.

Reply:
Good point. I will include this in a follow up PR.